base/include/ImageMetadata.h

```diff
@@ -24,7 +24,8 @@ class ImageMetadata
 		BAYERBG8, // BGGR
 		BAYERGB8, // GBRG
 		BAYERGR8, // GRBG
-		BAYERRG8  // RGGB
+		BAYERRG8,  // RGGB
+		YUYV
```

Comment:
Better to put them next to each other. we can leave holes, e.g. all Bayers can start at 20, all YU* can start at 10